vast.yaml.example

```diff
@@ -30,6 +30,10 @@ vast:
   # release builds.
   # file-verbosity: debug
 
+  # Whether to enable automatic log rotation. If set to true, a new log file
+  # will be created when the size of the current log file exceeds 10 MiB.
+  #log-rotation: true
```

Comment:
Is it possible to offer an option to configure the hard-coded 10 MiB limit? As a user, I would want to have this tuning knob right next to the on/off switch.

Reply:
It is of course possible, but this is not the only tuning knob that spdlog exposes:
 * file size threshold
 * number of rotated log files that should be kept
 * should we use size-based or time-based rotation?
 * for time-based rotation, the interval (daily/weekly/...) and starting point

I left all of this out for now. I'm happy to add a single additional option for the threshold if you want it, but I'd really like to avoid building a "general" solution for this.

Reply:
I was not saying that it should be in this PR, but I do think we should give the user a little more flexibility to pass options to spdlog. After all, a pass-through is not a lot of code complexity. Let's discuss what exactly we deem the minimal set of features to make users happy. I agree that we don't need to support the whole enchilada out of the box.

Reply:
Sounds good, I've created https://app.clubhouse.io/tenzir/story/22662/expose-more-spdlog-options-to-the-user as a follow-up story for that.

Reply:
Perfect, I think that's the right trade-off for now.